src/aics-image-viewer/components/App/index.tsx

```diff
@@ -215,15 +214,6 @@ const App: React.FC<AppProps> = (props) => {
   // To do this it requires access to the current state value, not the one it closed over)
   const [channelSettings, setChannelSettings, getChannelSettings] = useStateWithGetter<ChannelState[]>([]);
 
-  const getInitialAlphaLevel = (): number => {
-    if (props.viewerSettings?.maskAlpha !== undefined) {
-      return props.viewerSettings.maskAlpha;
```

Comment:
are we losing the ability to apply this setting now?

Reply:
`maskAlpha` will still respond to an initial value passed in via props, if that's the question. The point of this function was to derive a new value whenever something happened that reset mask alpha. But without it, the prop still gets applied in the `viewerSettings` initializer.